### PR TITLE
Cognito redirect ur ls configurable

### PR DESCRIPTION
**Pull request by stevbark**
Should fix the redirect issue when we update the URL value in AWS.

### CODE REVIEW

File: api/.env.example [hunk 1]
```diff
@@ -5,4 +5,5 @@ COGNITO_REDIRECT_URI=http://localhost:4040/signin
 COGNITO_USER_POOL_ID=
 SECRET_KEY=
 COGNITO_ACCESS_ID=
-COGNITO_ACCESS_KEY=
\ No newline at end of file
+COGNITO_ACCESS_KEY=
+URL=http://localhost:4040
```
Comment (paulespinosa) [inline api/.env.example:9]: A more descriptive name would help me understand the purpose of this variable.

**Comment by paulespinosa** [comment]
This change is for Issue #503, right?

**Comment by stevbark** [comment]
> This change is for Issue #503, right?

yes. https://github.com/hackforla/HomeUniteUs/issues/503

File: api/.env.example (hunk 1 above)
Comment (ju1es) [reply api/.env.example:9]: +1

**Comment by ju1es** [comment]
sorry in advance if I'm not understanding this correctly! but what's the purpose of `.env.example`? if it's the `.env` that will run in production, i think we'd specify environment variables in the docker file right? I started cooking up a config system that'll generate at runtime,

https://github.com/hackforla/HomeUniteUs/pull/527/files

maybe we add this there?

**Comment by paulespinosa** [comment]
@ju1es 
> but what's the purpose of `.env.example` ?

My understanding is that it's a file that we can check into version control (git) to demonstrate the environment variables that need to be set within an deployment environment without the actual values (secrets). 

> i think we'd specify environment variables in the docker file right?

If the `.env` file exists when building the docker image, it will be pulled into the docker image and used by the application when it starts up. I imagine the prod version of the `.env` file is stored somewhere in AWS that the continuous integration build can access. The CI process is something I'm not familiar with yet, though.

**Comment by stevbark** [comment]
New changes pushed. more descriptive name added.

**Pull request merged**